radar/2022-03-28/nocode-lowcode.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "No-Code / Low-Code"
+ring: assess
+quadrant: methods-and-patterns
+featured: true
+---
+
+No-code or low-code refers to tools that allow application software to be created via graphical user interfaces and configuration - instead of (classic) text-based programming.
+Low-code platforms additionally supports certain scripting possibilities.
+
+No-code and low-code platforms have raised a lot of funding and the ecosystem is growing fast. There are more than 400 platforms listed on software selection websites like g2.
+The promise of this ecosystem sounds logical: there is a growing demand for digital products and solutions in all sectors - and skilled teams of developers are scarce. 
+That's why platforms like bubble.io, airtable, appsheet, bryter, ninox and co promise to build digital products faster and better.
```

Comment:
Would you mind to add links to these sites?

Reply:
@resah sure - just did so